src/server/https/auth/routes.ts

```diff
@@ -1,8 +1,8 @@
 import { FastifyPluginCallback, FastifyPluginOptions } from 'fastify';
 import { Server } from 'http';
-import { authMeHandler } from './handler';
+import { authMeHandler } from '../../../auth/handler';
```

Comment:
Instead of using handler like this. Apply a technique called dependency injection to get auth functions inside server

Reply:
i Read their docs still i am not understanding anything on dependency injection